addons/web/static/src/less/kanban_view_mobile.less

```diff
@@ -29,7 +29,7 @@
             }
         }
 
-        &.o_kanban_small_column .o_kanban_group:not(.o_column_folded) {
+        .o_kanban_group:not(.o_column_folded) {
```

Comment:
is the rule enough specific to make the width: 100% rule apply (to override this one: https://github.com/odoo/odoo/blob/11.0/addons/web/static/src/less/kanban_view.less#L448)?

Reply:
you are right, I thought `@media` added specificity =/ I must have thought that because we usually have them after no media ones.

I prefixed by `&[class]` and test it when `o_kanban_small_column` was present.